### DISCUSSION TITLE
when start meta by docker,how to set param "single" or "join"

### DISCUSSION THREAD

**Discussion by WUWEI111222333** [databendlabs/databend#10825] [Q&A]
![image](https://user-images.githubusercontent.com/35459358/228527514-0dd582c6-9ea3-4ff7-803a-0b843533b57e.png)

**Comment by everpcpc** [comment]
If you would like to test databend locally, an all-in-one image is recommaned:
https://hub.docker.com/r/datafuselabs/databend

**Reply by WUWEI111222333** [reply]
i have deployed a all-in-one for test,now i want to deploy a cluster by docker for production

**Reply by everpcpc** [reply]
For production usage, a helm chart is recommanded:
https://github.com/datafuselabs/helm-charts
https://databend.rs/doc/deploy/deploying-databend-on-kubernetes

But if you want to deploy manually, you can refer to the argument in helm chart deployment:
https://github.com/datafuselabs/helm-charts/blob/main/charts/databend-meta/templates/statefulset.yaml#L60
 https://databend.rs/doc/deploy/metasrv/metasrv-deploy

**Reply by WUWEI111222333** [reply]
thanks
